safe-global/safe-wallet-web
# Feat: general app settings in the footer

## What it solves

Resolves #1994

## How this PR fixes it

Makes the Settings page work for not Account-specific URLs.

## How to test it
Case 1:
* Open Welcome
* Go to Settings via the footer

Case 2:
* Open a Safe
* Go to Safe settings via the sidebar menu

Case 3:
* Check that the Cookie banner still works and looks like before

## Screenshots
<img width="1248" alt="Screenshot 2023-05-17 at 12 48 59" src="https://github.com/safe-global/safe-wallet-web/assets/381895/b0e3364e-6d1e-4d68-a838-4f94fee03cf4">
